Problems/2798/ans_2798.py

```diff
@@ -0,0 +1,19 @@
+#https://www.acmicpc.net/problem/2798
+
+def solve():
+    N, M = input().split()
```

Comment:
```suggestion
    N, M = map(int,input().split())
```
int type으로 먼저 선정하면 좋을 것으로 보입니다.